library/src/morpheus-master/sass/modules/tool/sitemanage/_sitemanage.scss

```diff
@@ -678,6 +678,10 @@
     }
   }
 
+  .enrollmentStatus {
```

Comment:
just use bootstrap .small instead of adding new class?

Reply:
Aren't we trying to move away from using bootstrap, steadily?

Reply:
Shawn always gives me the impression that bootstrap is bad down the road.

Reply:
no idea 🤷

Reply:
I'm pretty sure that's the case, just trying to move away from our dependence on bootstrap for basic styling stuff. I've updated that class to use one of our variables.